### PR TITLE
Handling chemical formula larger than 30 chars

### PR DESCRIPTION
**Pull request by sudarsan-surendralal**
This is an intermediate fix till a proper solution is found.

### CODE REVIEW

File: pyiron_atomistics/atomistics/job/atomistic.py [hunk 1]
```diff
@@ -394,7 +394,10 @@ def db_entry(self):
                 parent_structure = self.structure.get_parent_basis()
             else:
                 parent_structure = self.structure.copy()
-            db_dict["ChemicalFormula"] = parent_structure.get_chemical_formula()
+            chem_formula = parent_structure.get_chemical_formula()
+            if len(chem_formula) > 30:
+                chem_formula = chem_formula[:29]+"+"
```
Comment (liamhuber) [inline pyiron_atomistics/atomistics/job/atomistic.py]: My concern with leaving any of the real formula intact is that then certain parsing operations will silently return the wrong results. e.g. imagine that our limits were much shorter and we would up with something like "Fe302Mn1000" -> "Fe302Mn10" (wrong count), or "O1234P56He" -> "O1234P56H" (wrong species).

I would suggest instead a value like "OVERFLOW_ERROR", and let the user exploit pyiron tables to extract the chemistry directly from the stored structures.

**Review by liamhuber — CHANGES_REQUESTED** [review]
Cool, nice and fast! I would suggest only to change the actual stored value to avoid silent errors.

**Comment by jan-janssen** [comment]
I do not think we should make cluster dependent bug fixes, as the issue is already fixed in https://github.com/pyiron/pyiron_base/pull/290 . If you want to modify the version on our cluster that is fine for me but I am against including this workaround in the master branch.

**Comment by sudarsan-surendralal** [comment]
I close this since @muh-hassani has already proposed a better solution in https://github.com/pyiron/pyiron_base/pull/295

**Pull request closed without merging**